### ISSUE TITLE
[New rule] EdgeInsets

### ISSUE THREAD

**Issue by NickBalnaves** [dart-code-checker/dart-code-metrics#743]
**Please describe what the rule should do:**
If 0 values are mostly passed into EdgeInsets.fromLTRB then it makes sense to use EdgeInsets.only for only the values that are to be used. 
Also if there are symmetric values passed into EdgeInsets.fromLTRB or EdgeInsets.only then it makes sense to convert this into the EdgeInsets.symmetric equivalent.
In general I also find EdgeInsets.fromLTRB less easy to read as opposed to EdgeInsets.only and EdgeInsets.symmetric so I would like the option to always change it to only or symmetric.

**What category of rule is this? (place an "X" next to just one item)**
[ ] Warns about a potential error (problem)
[x] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

Bad:
```dart
EdgeInsets.fromLTRB(8, 0, 8, 0)
EdgeInsets.fromLTRB(8, 0, 0, 0)
EdgeInsets.only(left: 16, right: 16)
EdgeInsets.fromLTRB(8, 8, 8, 8)
```

Good:
```dart
EdgeInsets.symmetric(horizontal: 8)
EdgeInsets.only(left: 8)
EdgeInsets.symmetric(horizontal: 16)
EdgeInsets.all(8)
```

**Are you willing to submit a pull request to implement this rule?**
No

**Comment by incendial** [comment]
Available in 4.17.0 🚀
